Components/Typography/index.html

```diff
@@ -0,0 +1,167 @@
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+          <div class="kash-p">
+            <h3 class="kash-h5">Customizing headings</h3>
+            <p class="kash-subtitle">
+              Use the included utility classes to recreate the small secondary
+              heading text
+            </p>
+          </div>
+          <div
+            class="kash-container kash-bg-light-gray kash-flex kash-flex-col kash-justify-center kash-align-center kash-border-sm kash-m kash-p kash-gap"
+          >
+            <h3 class="kash-h4">
+              Fancy display heading
+              <small class="kash-text-light">With faded secondary text</small>
+            </h3>
+          </div>
+          <div class="kash-code-container">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=htmlmixed&width=779&ds=true&dsyoff=4px&dsblur=12px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=12px&lh=158%25&si=false&es=2x&wm=false&code=%253Ch3%2520class%253D%2522kash-h4%2522%253E%250A%2520%2520Fancy%2520display%2520heading%250A%2520%2520%253Csmall%2520class%253D%2522kash-text-light%2522%253EWith%2520faded%2520secondary%2520text%253C%252Fsmall%253E%250A%253C%252Fh3%253E%250A"
+              style="
+                width: 100%;
+                height: 38vh;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+          <div class="kash-p">
+            <h3 class="kash-h5">Customizing text</h3>
+            <p class="kash-subtitle">styling common inline elements</p>
+          </div>
+          <div
+            class="kash-container kash-bg-light-gray kash-flex kash-flex-col kash-justify-center kash-align-center kash-border-sm kash-m kash-p kash-gap"
+          >
+            <p>
+              You can use the mark tag to
+              <mark class="kash-highlight">highlight</mark> text.
+            </p>
+            <p>
+              <del
+                >This line of text is meant to be treated as deleted text.</del
+              >
+            </p>
+            <p>
+              <s
+                >This line of text is meant to be treated as no longer
+                accurate.</s
+              >
+            </p>
+            <p>
+              <ins
+                >This line of text is meant to be treated as an addition to the
+                document.</ins
+              >
+            </p>
+            <p><u>This line of text will render as underlined.</u></p>
+            <p>
+              <small
+                >This line of text is meant to be treated as fine print.</small
+              >
```

Comment:
Use an extension like prettier or beautify to reformat indentation and code before pushing

Reply:
I have used prettier extension extension only auto saved like this